django/db/backends/base/references.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import unicode_literals
+
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
+class TableReference(object):
+    def __init__(self, table, prepare):
```

Comment:
Some comments would be useful. It's not obvious what `prepare` is -- seems to be a function of some sort. Maybe even a module docstring explaining why all this stuff exists.